<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlRoomDialog</class>
 <widget class="QDialog" name="ControlRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="7">
    <widget class="QPushButton" name="dmp3_button">
     <property name="text">
      <string>Damper 3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="fs_label">
     <property name="text">
      <string>Fan Speed: 0%</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" rowspan="3">
    <widget class="QDial" name="csp_dial">
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>600</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4" rowspan="3">
    <widget class="QDial" name="fs_dial">
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="3">
    <widget class="QDial" name="dsp_dial">
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>600</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="5" rowspan="3">
    <widget class="QDial" name="ds_dial">
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="ch_label">
     <property name="text">
      <string>Cat Heat: 0%</string>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QPushButton" name="door_button">
     <property name="text">
      <string>Door</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" rowspan="3">
    <widget class="QDial" name="ch_dial">
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="7">
    <widget class="QPushButton" name="cooler_button">
     <property name="text">
      <string>Cooler</string>
     </property>
    </widget>
   </item>
   <item row="10" column="7">
    <widget class="QPushButton" name="dmp4_button">
     <property name="text">
      <string>Damper 4</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QPushButton" name="stir_button">
     <property name="text">
      <string>Stir</string>
     </property>
    </widget>
   </item>
   <item row="8" column="7">
    <widget class="QPushButton" name="ex_button">
     <property name="text">
      <string>Exhaust</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="ds_label">
     <property name="text">
      <string>Drum Speed: 0%</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="csp_label">
     <property name="text">
      <string>Cat Set Point: 100F</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dsp_label">
     <property name="text">
      <string>Drum Set Point: 100F</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="3">
    <widget class="QDial" name="dh_dial">
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="dh_label">
     <property name="text">
      <string>Drum Heat: 0%</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QWidget" name="graph_widget" native="true"/>
   </item>
   <item row="1" column="0" rowspan="2" colspan="8">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="6" rowspan="9">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
